chance/p5js/chance-contrast/index.md

```diff
@@ -0,0 +1,9 @@
+layout: p5js
+author: Edmundo Mejía Galindo
+author-url: https://github.com/edgalindo/
+title: Law of Casuality
```

Comment:
Titels should not be the name of a law. If you want to introduce a new law you need to start to write a definition.